public/scripts/Form.js

```diff
@@ -70,6 +70,25 @@ TextInput.propTypes = {
   label: PropTypes.string
 }
 
+export var TextareaInput = (props) => {
+ let textareaInputStyle = {
+   paddingTop: 10,
+  }
+
+  return (
+    <div className="form-group" style={textareaInputStyle}>
+      <label>{props.label}</label>
+      <textarea className="form-control"
+                value={props.value || ''} {...props}
+                rows="10" cols="50"/>
```

Comment:
This should probably come in from the parent, so leave that blank here and specify as prop where you instantiate the component